run.py

```diff
@@ -4,6 +4,7 @@
 import datetime
 import time
 import simplejson
+from libs.build_params import r0, OUTPUT_DIR, INTERVENTIONS
```

Comment:
I would recommend just importing the module:
`from libs import build_params`

it's helpful for keeping track of where variables come from

Reply:
Done.